src/ximea_driver_node.cpp

```diff
@@ -4,7 +4,8 @@ using namespace trailbot;
 
 int main(int argc, char** argv) {
   rclcpp::init(argc, argv);
-  rclcpp::spin(std::make_shared<XIMEADriver>());
```

Comment:
Interesting that cpp ros node does not need spinning

Reply:
There aren't any callbacks, so no need to spin